packages/blockchain-wallet-v4/src/remote/types.ts

```diff
@@ -19,11 +19,8 @@ const cata = function <E, A>(
     case 'RemoteFailure': {
       return obj.Failure(this.error)
     }
-    case 'RemoteSuccess': {
-      return obj.Success(this.data)
-    }
     default:
-    // no-action
+      return obj.Success(this.data)
```

Comment:
There is `notReachable` util here: https://github.com/blockchain/blockchain-wallet-v4-frontend/pull/5531/files#diff-ac10f9c3be5e1cc5f70b9486f99b45fd3e937fdf0d7672cd74f74a1d9b627f72

maybe we should use it here as well after it's merged